pepys_import/core/store/data_store.py

```diff
@@ -705,6 +705,10 @@ def get_sensors_by_platform_type(self, platform_type):
             .all()
         )
 
+    def get_states(self):
+        # get list of all states in the DB
+        return self.session.query(self.db_classes.State).all()
```

Comment:
I had _no idea_ how to do this :-)

Reply:
Really? I wish I had implemented it before :)

Reply:
I know _practically nothing_ about SQLAlchemy.

In recent weeks I've learned _a tiny bit more than that_ about Python.